antidox/perspective.py

```diff
@@ -75,58 +77,90 @@ def dlp_request(dlp, apikey_data, comment):
           "includeQuote":True
           }
       }
-  dlp_response = (dlp.projects().content().inspect(body=dlp_request,
+  dlp_response = (dlp.projects().content().inspect(body=request_dlp,
                                                    parent='projects/'+
                                                    apikey_data['project_number']
                                                    ).execute())
   return dlp_response
 
 
-# Checking/returning comments that are likely or very likely to contain PII
 def contains_pii(dlp_response):
-  """ specifically checks the dlp response for results containing PII"""
+  """ Checking/returning comments that are likely or very likely to contain PII
+
+      Args:
+      passes in the resukts from the cloud DLP
+      """
   has_pii = False
   if 'findings' not in dlp_response['result']:
     return False, None
   for finding in dlp_response['result']['findings']:
     if finding['likelihood'] in ('LIKELY', 'VERY_LIKELY'):
-      # print('finding:', finding['infoType'], finding['likelihood'])
       has_pii = True
       return (has_pii, finding['infoType']["name"])
   return False, None
 
 
-# Checking/returning comments with a toxicity value of over 50 percent.
 def contains_toxicity(perspective_response):
-  """ specifically checks the perspective response for toxicity score"""
+  """Checking/returning comments with a toxicity value of over 50 percent."""
   is_toxic = False
   if (perspective_response['attributeScores']['TOXICITY']['summaryScore']
       ['value'] >= .5):
     is_toxic = True
   return is_toxic
 
 
+def get_wikipage(pagename):
+  """ Gets all content from a wikipedia page and turns it into plain text. """
+  # pylint: disable=fixme, line-too-long
+  page = ("https://en.wikipedia.org/w/api.php?action=query&prop=revisions&rvprop=content&format=json&formatversion=2&titles="+(pagename))
+  get_page = requests.get(page)
+  response = json.loads(get_page.content)
+  text_response = response['query']['pages'][0]['revisions'][0]['content']
+  text = clean.content_clean(text_response)
+  return text
+
+# pylint: disable=fixme, too-many-locals
 def main(argv):
-  parser = argparse.ArgumentParser()
+  """ runs dlp and perspective on content passed in """
+  parser = argparse.ArgumentParser(description='Process some integers.')
   parser.add_argument('--input_file', help='Location of file to process')
   parser.add_argument('--api_key', help='Location of perspective api key')
+  parser.add_argument('--sql_query',)
+  parser.add_argument('--csv_file')
+  parser.add_argument('--wiki_pagename')
```

Comment:
Can you add a help string to this, similar to the --input_file and --api_key args?  TJ recently added them for --sql_query and --csv_file so you could take his strings from #193